src/app-sandbox/index.tsx

```diff
@@ -4,6 +4,7 @@ import { Store } from 'redux';
 
 import { Apps } from '../lib/apps';
 import { App as FeedApp } from '@zer0-os/zos-feed';
+import StakingApp from '@zero-tech/zapp-staking';
```

Comment:
maybe worth dropping the default export, but not the end of the world since this is the only usage.

Reply:
Good point - I'll drop it for consistency!